src/detectors/EEMC/CalorimeterHit_factory_EcalEndcapNRecHits.h

```diff
@@ -26,15 +26,15 @@ class CalorimeterHit_factory_EcalEndcapNRecHits : public JFactoryT<edm4eic::Calo
         m_input_tag = "EcalEndcapNRawHits";
 
         // digitization settings, must be consistent with digi class
-        m_capADC=8096;//{this, "capacityADC", 8096};
-        m_dyRangeADC=100. * MeV;//{this, "dynamicRangeADC", 100. * MeV};
-        m_pedMeanADC=400;//{this, "pedestalMean", 400};
-        m_pedSigmaADC=3.2;//{this, "pedestalSigma", 3.2};
+        m_capADC=16384;//{this, "capacityADC", 8096};
+        m_dyRangeADC=20. * GeV;//{this, "dynamicRangeADC", 100. * MeV};
+        m_pedMeanADC=100;//{this, "pedestalMean", 400};
+        m_pedSigmaADC=1;//{this, "pedestalSigma", 3.2};
         m_resolutionTDC=10 * dd4hep::picosecond;//{this, "resolutionTDC", 10 * ps};
 
         // zero suppression values
         m_thresholdFactor=4.0;//{this, "thresholdFactor", 0.0};
-        m_thresholdValue=0.0;//{this, "thresholdValue", 0.0};
+        m_thresholdValue=3.0;//{this, "thresholdValue", 0.0};
```

Comment:
Just out of curiosity, what is the unit of m_thresholdValue?

Reply:
It seems in units of ADC.